### PR TITLE
Add CSS for developer settings pages

### PR DESCRIPTION
**Pull request by rfriberg**
Add CSS for developer settings pages

### CODE REVIEW

File: src/stylesheets/project_specific/developer/_developer-settings.scss [hunk 1]
```diff
@@ -4,4 +4,19 @@
     margin-right: 12px;
   }
 
+  .plan .free {
```
Comment (louh) [inline src/stylesheets/project_specific/developer/_developer-settings.scss:7]: I'm generally really antsy with class names like `.plan .free`. Even though it's nested here for this page, someone else might come along and make a top-level component class `.plan` and suddenly it collides with the one here.

Also, this only `@extend`s one class name, and it's very much related to the toggle switch, I would imagine that it's better to alter the markup to use the `.toggle-switch-text-disabled/enabled` class names directly?

File: src/stylesheets/project_specific/developer/_forms.scss [hunk 2]
```diff
@@ -1,5 +1,85 @@
...
-  /* Braintree-specific class */
-  .field_with_errors {
-    @extend .has-error;
-  }
+/* Braintree-specific class */
+.field_with_errors {
+  @extend .has-error;
+}
+
+.dollah-sign {
```
Comment (louh) [inline src/stylesheets/project_specific/developer/_forms.scss:7]: What is `.dollah-sign` referring to?

File: src/stylesheets/project_specific/developer/_forms.scss [hunk 3]
```diff
@@ -1,5 +1,85 @@
...
+  line-height: 40px; // to match input field height
+}
+
+input[disabled],
+input[disabled] + .dollah-sign {
+  color: #ccc;
+}
+
+// Toggle Switch
+// Should we move to common/_forms partial?
```
Comment (louh) [inline src/stylesheets/project_specific/developer/_forms.scss:21]: Definitely seems like a reusable component

File: src/stylesheets/project_specific/developer/_forms.scss [hunk 4]
```diff
@@ -1,5 +1,85 @@
...
+
+.toggle-switch {
+  position: relative;
+  display: inline-block;
+  width: $toggle-width;
+  height: $toggle-height;
+}
+
+// Hide default HTML checkbox
+.toggle-switch input {display:none;}
```
Comment (louh) [inline src/stylesheets/project_specific/developer/_forms.scss]: I would prefer to see this split onto multiple lines for consistency. e.g.
```
.toggle-switch input {
  display: none;
}
```

File: src/stylesheets/project_specific/developer/_forms.scss [hunk 5]
```diff
@@ -1,5 +1,85 @@
...
+
+.toggle-switch .slider {
```
Comment (louh) [inline src/stylesheets/project_specific/developer/_forms.scss]: Do we use the `.slider` class anywhere else? would this name ever be confused with the range slider? (I would name `.slider` `.toggle-switch-slider` to make it clear what component this is attached to.)

File: src/stylesheets/project_specific/developer/_developer-settings.scss (hunk 1 above)
Comment (rfriberg) [reply src/stylesheets/project_specific/developer/_developer-settings.scss:7]: Ordinarily, I'd agree with that suggestion.  But `plan`, `free`, and `metered` are all tightly coupled to both the rails app and the javascript used for toggling the spending limit field.  Adding classes (which are name spaced to this section of the developer app anyway) is preferable to making updates to the app/js logic at this point.

File: src/stylesheets/project_specific/developer/_forms.scss (hunk 2 above)
Comment (rfriberg) [reply src/stylesheets/project_specific/developer/_forms.scss:7]: A `$` component we're using in the form.

File: src/stylesheets/project_specific/developer/_forms.scss (hunk 3 above)
Comment (rfriberg) [reply src/stylesheets/project_specific/developer/_forms.scss:21]: Yep.  It's available to all developer forms right now.  I want to check with Ekta, though, before deciding that this should be more widely available.

File: src/stylesheets/project_specific/developer/_forms.scss (hunk 4 above)
Comment (rfriberg) [reply src/stylesheets/project_specific/developer/_forms.scss]: 👍  Updating now.

File: src/stylesheets/project_specific/developer/_forms.scss (hunk 5 above)
Comment (rfriberg) [reply src/stylesheets/project_specific/developer/_forms.scss]: We don't use it, but that's a good point.  I'm going to add the `.toggle-switch-slider` class but won't remove the `.slider` until the updates are made on developer repo, just to minimize the back-and-forth right before a launch.

File: src/stylesheets/project_specific/developer/_forms.scss (hunk 2 above)
Comment (louh) [reply src/stylesheets/project_specific/developer/_forms.scss:7]: Did we make that or is it a Braintree thing?

**Review by louh — APPROVED** [review]

**Pull request merged**